src/scripts/getTemp.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/prefer-default-export */
+async function getIp() {
```

Comment:
- [ ] As mentioned in the comment above, let's use arrow functions in the functions you have here.